src/sagemaker/session.py

```diff
@@ -3599,6 +3649,19 @@ def create_model_package_from_containers(
                 "IMAGE_CLASSIFICATION", "OBJECT_DETECTION", "TEXT_GENERATION", "IMAGE_SEGMENTATION",
                 "CLASSIFICATION", "REGRESSION", "OTHER" (default: None).
         """
+        if containers:
+            # Containers are provided. Now we can merge missing entries from config.
+            # If Containers are not provided, it is safe to ignore. This is because,
+            # if this object is provided to the API, then Image is required for Containers.
+            # That is not supported by the config now. So if we merge values from config,
+            # then API will throw an exception. In the future, when SageMaker Config starts
+            # supporting other parameters we can add that.
```

Comment:
If this this removed we should still have the same behavior right? (because the helper will skip injecting if Image is not present.) 

Not a huge deal but nice to keep simple